lk2nd/device/dts/msm8952/msm8917-huawei-agassi.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+
+/*
+ * This device is quirky. It excluded from generic build.
+ * To run lk2nd on this device you need to flash custom bootloader and unlock it. Read more https://wiki.postmarketos.org/wiki/Huawei_Mediapad_T3_10_(huawei-agassi)#Bootloader_unlocking.
+ *
+ * Compile lk2nd with the following flag to run on this device
+ * LK2ND_ADTBS="msm8917-huawei-agassi.dtb"
+ */
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8917 0>;
+	qcom,board-id = <0x0001000b 0x00>;
+};
+
+&lk2nd {
+	/* 
+	 * There are 3 versions: 
+	 *  - L-09 (Global)
+	 *  - L-03 (USA)
+	 *  - W-09 (WiFi only)
+	 */
+
+	huawei-agassi {
+		model = "Huawei MediaPad T3 10 (agassi)";
+		compatible = "huawei,agassi";
+		lk2nd,dtb-files = "msm8917-huawei-agassi";
+		lk2nd,match-cmdline = "*androidboot.bootloader.version=P2200_DEFAULT_BOOTLOADER_170523*";
```

Comment:
I'm actually not sure if this is useful, perhaps could drop this and use LK2ND_COMPATIBLE="huawei,agassi" at build time additionally? This would force lk2nd to always find this node